carma_wm/include/carma_wm/WMTestLibForGuidance.h

```diff
@@ -0,0 +1,458 @@
+  auto pt34 = carma_wm::test::getPoint(3 * width,4 * length, 0);
+
+  lanelet::LineString3d ls00(lanelet::utils::getId(), {pt00,pt01} );
+  lanelet::LineString3d ls01(lanelet::utils::getId(), {pt01,pt02} );
+  lanelet::LineString3d ls02(lanelet::utils::getId(), {pt02,pt03} );
+  lanelet::LineString3d ls03(lanelet::utils::getId(), {pt03,pt04} );
+
+  lanelet::LineString3d ls10(lanelet::utils::getId(), {pt10,pt11} );
+  lanelet::LineString3d ls11(lanelet::utils::getId(), {pt11,pt12} );
+  lanelet::LineString3d ls12(lanelet::utils::getId(), {pt12,pt13} );
+  lanelet::LineString3d ls13(lanelet::utils::getId(), {pt13,pt14} );
+
+  lanelet::LineString3d ls20(lanelet::utils::getId(), {pt20,pt21} );
+  lanelet::LineString3d ls21(lanelet::utils::getId(), {pt21,pt22} );
+  lanelet::LineString3d ls22(lanelet::utils::getId(), {pt22,pt23} );
+  lanelet::LineString3d ls23(lanelet::utils::getId(), {pt23,pt24} );
+
+  lanelet::LineString3d ls30(lanelet::utils::getId(), {pt30,pt31} );
+  lanelet::LineString3d ls31(lanelet::utils::getId(), {pt31,pt32} );
+  lanelet::LineString3d ls32(lanelet::utils::getId(), {pt32,pt33} );
+  lanelet::LineString3d ls33(lanelet::utils::getId(), {pt33,pt34} );
+
+  all_lanelets.push_back(getLanelet(1200, ls00,ls10,lanelet::AttributeValueString::Solid, lanelet::AttributeValueString::Dashed));
+  all_lanelets.push_back(getLanelet(1201, ls01,ls11,lanelet::AttributeValueString::Solid, lanelet::AttributeValueString::Dashed));
+  all_lanelets.push_back(getLanelet(1202, ls02,ls12,lanelet::AttributeValueString::Solid, lanelet::AttributeValueString::Dashed));
+  all_lanelets.push_back(getLanelet(1203, ls03,ls13,lanelet::AttributeValueString::Solid, lanelet::AttributeValueString::Dashed));
+
+  all_lanelets.push_back(getLanelet(1210, ls10,ls20,lanelet::AttributeValueString::Dashed, lanelet::AttributeValueString::Dashed));
+  all_lanelets.push_back(getLanelet(1211, ls11,ls21,lanelet::AttributeValueString::Dashed, lanelet::AttributeValueString::Dashed));
+  all_lanelets.push_back(getLanelet(1212, ls12,ls22,lanelet::AttributeValueString::Dashed, lanelet::AttributeValueString::Dashed));
+  all_lanelets.push_back(getLanelet(1213, ls13,ls23,lanelet::AttributeValueString::Dashed, lanelet::AttributeValueString::Dashed));
+  
+  all_lanelets.push_back(getLanelet(1220, ls20,ls30,lanelet::AttributeValueString::Dashed, lanelet::AttributeValueString::Solid));
+  all_lanelets.push_back(getLanelet(1221, ls21,ls31,lanelet::AttributeValueString::Dashed, lanelet::AttributeValueString::Solid));
+  all_lanelets.push_back(getLanelet(1222, ls22,ls32,lanelet::AttributeValueString::Dashed, lanelet::AttributeValueString::Solid));
+  all_lanelets.push_back(getLanelet(1223, ls23,ls33,lanelet::AttributeValueString::Dashed, lanelet::AttributeValueString::Solid));
+  
+  // Create basic map and verify that the map and routing graph can be build, but the route remains false	
+  lanelet::LaneletMapPtr map = lanelet::utils::createMap(all_lanelets, {});
```

Comment:
Run MapConformer.ensureCompliance(map);
On this before returning it. That will ensure the map matches the setup of an actual carma map.